dali/pipeline/operators/color_space/color_space_conversion.h

```diff
@@ -30,6 +32,10 @@ class ColorSpaceConversion : public Operator<Backend> {
   }
 
  protected:
+  bool SetupImpl(std::vector<OutputDesc> &output_desc, const workspace_t<Backend> &ws) override {
```

Comment:
API question:
Do we need the return value here? We already have `CanInferOutputs` - is it even legal for this function to return false if we declared that we can infer outputs? And, conversely, what happens if this function returns `true` when the operator declares that it can't infer outputs?

Reply:
In the first case I think it should be illegal. In the second case, that would probably result in inefficient use of `TensorVector`.
I can DALI_ENFORCE it, but I am not sure if we want to store the result of `CanInferOutputs` and compare it to all later invocations.

Reply:
I added the enforces but kept it as separate commit.